render/root.go

```diff
@@ -9,22 +9,20 @@ import (
 	"github.com/tidbyt/gg"
 )
 
-const (
-	// DefaultFrameWidth is the normal width for a frame.
-	DefaultFrameWidth = 64
+// DefaultFrameWidth is the normal width for a frame.
+var DefaultFrameWidth = 64
```

Comment:
Could you leave these as constants?

Reply:
So the issue was that [`marquee.go`](https://github.com/tidbyt/pixlet/blob/main/render/marquee.go#L78-L82) uses these constants.

I can leave these as constants and then create :

``` golang
var FrameWidth = DefaultFrameWidth
var FrameHeight = DefaultFrameHeight
```

which I could then use in marquee.go:

```
	if m.isVertical() {
		cb = m.Child.PaintBounds(image.Rect(0, 0, FrameWidth, m.Height*10), 0)
		cw = cb.Dy()
		size = m.Height
	} else {
		cb = m.Child.PaintBounds(image.Rect(0, 0, m.Width*10, FrameHeight), 0)
		cw = cb.Dx()
		size = m.Width
	}
```

Would that work?

Reply:
Yes, this works! We may want to clean this up later by passing this information into the render tree, but we can follow up in a future change for something like that.